packages/sampler/stories/qa/VirtualGridList.js

```diff
@@ -60,6 +64,102 @@ const updateDataSize = (dataSize) => {
 
 updateDataSize(defaultDataSize);
 
+let itemList = [];
+for (let i = 0; i < 60; i++) {
+	itemList.push('item' + i);
+}
+
+const ContexturePopupButton = ContexturePopupDecorator(Button);
+
+let lastIndex = 0;
+
+class MyVirtualList extends React.Component {
+	componentDidMount () {
+		this.scrollTo({index: lastIndex, animate: false, focus: true});
+	}
+
+	closePopup (index) {
+		lastIndex = index;
+		// eslint-disable-next-line enact/prop-types
+		this.props.closePopup();
+	}
+
+	renderItem = ({index, ...rest}) => {
+		return (
+			/* eslint-disable react/jsx-no-bind */
+			<Item key={index} onClick={() => this.closePopup(index)} {...rest}>{itemList[index]}</Item>
+		);
+	};
+
+	getScrollTo = (scrollTo) => {
+		this.scrollTo = scrollTo;
+	}
+
+	render () {
+		let props = {...this.props};
+		delete props.closePopup;
+
+		return (
+			<div {...props} style={{width: '915px', height: '600px'}}>
+				<VirtualGridList
+					dataSize={itemList.length}
+					itemRenderer={this.renderItem}
+					itemSize={{minWidth: ri.scale(285), minHeight: ri.scale(60)}}
+					direction="vertical"
+					spacing={ri.scale(0)}
+					cbScrollTo={this.getScrollTo}
+				/>
+			</div>
+		);
+	}
+}
+
+class ButtonAndVirtualGridList extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			isPopup: false
+		};
+	}
+
+	renderPopup = ({...rest}) => {
```

Comment:
can just use `(rest)` here, no need to destructure.

Reply:
Done